### DISCUSSION TITLE
posts/blog-optimization-again/

### DISCUSSION THREAD

**Discussion by giscus** [WincerChan/blog-comments#9] [Announcements]
# posts/blog-optimization-again/

好久没写博客了，每当我有些想写代码的情绪无处发泄的时候，我就会开始折腾之前写的老项目，而我的博客又是一个稍微有一点复杂的项目（Hugo + SolidStart），项目的复杂度一旦升高，难免会在不同需求之间做权衡，难以做到尽善尽美，所以博客一般是我折腾次数最多的。这次折腾的起因是我发现 SolidStart 在前段时间发布了 1.0 版本，相比之前的 beta 版本有不少改动，借着更新博客的机会，正好水一篇文章:)

https://blog.itswincer.com/posts/blog-optimization-again/

**Comment by hoosin** [comment]
很好， ***富哥***

**Reply by WincerChan** [reply]
日你
